Tests/CMakeLists.txt

```diff
@@ -251,6 +251,9 @@ add_test_rv(eb-c9 EB-C9)
 add_test_r(eb-c10 EB-C10)
 add_test_rv(eb-c11 EB-C11)
 add_test_rv(eb-c12 EB-C12)
+if(PELE_DIM EQUAL 2)
+    add_test_r(EB-FlowPastCylinder EB-FlowPastCylinder)
```

Comment:
This will fail. the first argument to the function should be the name of the input file. No need to run both Re cases as tests, so just change this to `add_test_r(EB-FlowPastCylinder-Re500 EB-FlowPastCylinder)`

Reply:
Maybe the established convention isn't great to begin with, but I would also ask that lowercase be used for the test name.